### PR TITLE
Create route and style

### PR DESCRIPTION
**Pull request by blueberry1312**
## For this PR, I do the following changes:

- Added Home, Calculator and Quote pages.
- Used react route to routing the three pages. 
- Styled all webapp.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,53 @@
...
+              </li>
+              <li>
+                <Link to="/calculator">Calculator</Link>
+              </li>
+              <li>
+                <Link to="/quote">Quote</Link>
+              </li>
+            </ul>
+          </nav>
+        </div>
```
Comment (VICTOR-CHUKWU) [inline src/App.js:36]: - [ ] you know you can actually make this into a component and import it here right? :+1:

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,53 @@
...
+        <Header />
+        <main>
+          <Routes>
+            <Route path="/" element={<Home />} />
+            <Route path="/calculator" element={<Calculator />} />
+            <Route path="/quote" element={<Quote />} />
```
Comment (VICTOR-CHUKWU) [inline src/App.js:42]: - [x] [OPTIONAL] you can add styling to your quote page, you can add some padding to the page, give the paragraph width, centre your text etc. just make it look nice 
Screenshot from 2023-03-28 20-14-29

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @blueberry1312 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed
- [ ] good use of React Router

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (blueberry1312) [reply src/App.js:36]: Yes! thanks for the feedback

**Pull request merged**